Testscripts/Windows/VERIFY-XDP-MULTIPLE-NICS.ps1

```diff
@@ -69,21 +67,13 @@ function Main {
         Write-LogInfo "  Internal IP : $($serverVMData.InternalIP)"
 
         # Check for compatible kernel
-        $currentKernelVersion = Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort `
-                -username $user -password $password -command "uname -r"
-        # ToDo: Update Minimum kernel version check once patches are in downstream distro.
-        if ($global:DetectedDistro -eq "UBUNTU"){
-            if ((Compare-KernelVersion $currentKernelVersion $MIN_KERNEL_VERSION) -lt 0){
-                Write-LogInfo "Unsupported kernel version: $currentKernelVersion"
-                return $global:ResultSkipped
-            }
-        } elseif ($global:DetectedDistro -eq "REDHAT"){
-            if ((Compare-KernelVersion $currentKernelVersion $RHEL_MIN_KERNEL_VERSION) -lt 0){
-                Write-LogInfo "Unsupported kernel version: $currentKernelVersion"
-                return $global:ResultSkipped
-            }
+        $numberOfXDPQueues = Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort `
+                -username $user -password $password -command "ethtool -S $iFaceName  | grep xdp_drop | wc -l"
+        Write-LogInfo "Number of XDP DROP queues: $numberOfXDPQueues"
+        if ($numberOfXDPQueues -gt 0) {
+            Write-LogInfo "Kernel version supports XDP"
         } else {
-            Write-LogInfo "Unsupported distro: $($global:DetectedDistro)."
+            Write-LogWarn "Unsupported Kernel $($global:initialkernelversion) or unsupported distro $($global:DetectedDistro)"

```

Comment:
Can we just print ```XDP not supported``` ? Same for all occurrences

Reply:
Giving kernel version is a valid point as this is a new feature and most of the distro version does not have it, mostly we have to run this with -CustomKernel parameter and to check which kernel version we have ran compatibility check on, it gives output right away.